src/commands/QueueCommand.ts

```diff
@@ -14,7 +14,7 @@ export class QueueCommand extends BaseCommand {
     @isMusicPlaying()
     public execute(message: IMessage): any {
         const embed = createEmbed("info")
-            .setTitle("**Song Queue**")
+            .setTitle("Song Queue")
```

Comment:
```suggestion
            .setTitle("Music Queue")
```
I better be renaming all the variables from song to music too, lol

Reply:
Oh my